packages/hooks/src/useFiatAccountList.ts

```diff
@@ -4,9 +4,7 @@ const useFiatAccountList = () => {
     const { client } = useStore();
     const { account_list, is_crypto } = client;
 
-    const fiat_account_list = account_list.filter(
-        account => !account.is_virtual && !is_crypto(account.title || '') && account.loginid?.startsWith('CR')
-    );
+    const fiat_account_list = account_list.filter(account => !account.is_virtual && !is_crypto(account.title || ''));
```

Comment:
removed the check for CR since when user has 1 MF fiat account, the list would return empty as it will be true only for CR accounts. 
then `fiat_account_list` will be empty
`fiat_account_list[0]` will be undefined